frontend/rust-lib/event-integration/tests/document/local_test/test.rs

```diff
@@ -153,3 +155,36 @@ async fn convert_document_test() {
     assert_eq!(result.html.is_some(), *html_assert);
   }
 }
+
+/// test convert data to json
+/// - input html: <p>Hello</p><p> World!</p>
+/// - input plain text: Hello World!
+/// - output json: { "type": "page", "data": {}, "children": [{ "type": "paragraph", "children": [], "data": { "delta": [{ "insert": "Hello" }] } }, { "type": "paragraph", "children": [], "data": { "delta": [{ "insert": " World!" }] } }] }
+#[tokio::test]
+async fn convert_data_to_json_test() {
+  let test = DocumentEventTest::new().await;
+  let _ = test.create_document().await;
+
+  let html = r#"<p>Hello</p><p> World!</p>"#;
+  let payload = ConvertDataToJsonPayloadPB {
+    data: html.to_string(),
+    input_type: InputType::Html,
+  };
+  let result = test.convert_data_to_json(payload).await;
+  let expect_json = json!({ "type": "page", "data": {}, "children": [{ "type": "paragraph", "children": [], "data": { "delta": [{ "insert": "Hello" }] } }, { "type": "paragraph", "children": [], "data": { "delta": [{ "insert": " World!" }] } }] });
```

Comment:
I am curious about why this json! is not auto format. You can try `cargo fmt --all` in the rust-lib directory